model_analyzer/plots/detailed_plot.py

```diff
@@ -135,13 +143,22 @@ def plot_data(self):
         on this plot's Axes object
         """
 
-        # Sort the data by concurrency
-        concurrency_sort_indices = list(
-            zip(*sorted(enumerate(self._data['concurrency']),
-                        key=lambda x: x[1])))[0]
+        # Need to change the default x-axis plot title for request rates
+        if 'request_rate' in self._data:
+            self._ax_latency.set_xlabel('Client Request Rate')
+
+        # Sort the data by request rate or concurrency
```

Comment:
Is this working? The image in the PR seems to be sorted by largest to smallest latency (?)

Reply:
The table is sorted by latency. The graph is sorted by concurrency/request rate.

Reply:
Can we file a story to fix the sorting of the table? Reverse latency seems wrong.  Either increasing concurrency/req_rate, or decreasing throughput.

Reply:
I've filed a story for this.